src/components/labels/Label.jsx

```diff
@@ -31,13 +35,15 @@ const Label = ({text, number, iconType, iconColor,
     <div className="sg-label__icon">
       <Icon type={iconType} color={iconColor} size={size.ICON_SIZE}/>
     </div>
-    <div className="sg-label__text">{text}</div>
+    {textElement}
     {numberElement}
+    {children}
   </div>;
 };
 
 Label.propTypes = {
-  text: PropTypes.string.isRequired,
```

Comment:
In action list there's an example with <Text> component, not with `sg-label__text` class